app/views/templates/base.php

```diff
@@ -21,41 +21,64 @@
     $css_include = $base_css;
 }
 
-$links = '
+/*
+    This is a closure to build a list item containing the
+    link to the page. The page we are on gets a CSS class
+    of selected_view added.
+*/
+$li_link = function ($page, $title, $text) use ($url, $urls) {
+    $link = array_search($page, $urls);
+
+    return '<li><a ' . ($url['path'] == $link ? 'class="selected_view" ' : '')
+           . 'href="/' . str_replace('/', '', $link) . '" '
```

Comment:
The only case where `$link` contains a slash is root, maybe it would clearer to just examine that case before the return statement and use `$link` directly
```PHP
$link = array_search($page, $urls);
if ($link != '/') {
    $link = "/{$link}";
}
```